h/search/__init__.py

```diff
@@ -24,6 +24,21 @@
 
 def includeme(config):
     settings = config.registry.settings
+
+    # Connection to version 6.x of ES follows
```

Comment:
Sooooo...I don't know offhand how to test this code directly, as it happens during app bootstrap. I'm happy to do what needs doing to make this more testable, whether that includes factoring out some of this logic into a separate function or something...? I have tests for `connect` but not for the `includeme`'s use of it, in other words.

Reply:
Especially since the config parsing (default values) can get moved into the config code once that code is refactored, I think its probably fine for this `includeme()` to not have any tests. It should hardly have any logic left in it. Maybe just the `if` around `client_poolsize`? Other than that if it was broken I think we'd know because h either wouldn't be starting up or wouldn't be functioning at all (no search working) in either dev or prod. There would be other tests that would be failing I think